mirdata/acousticbrainz_genre.py

```diff
@@ -0,0 +1,3800 @@
+                    0.105961687863,
+                    0.0136956917122,
+                    0.00351257342845,
+                    0.00198904867284
+                 ],
+                 'min':[
+                    0,
+                    0,
+                    0,
+                    0,
+                    0,
+                    0
+                 ],
+                 'dvar2':[
+                    0.0780283510685,
+                    0.0940517783165,
+                    0.0547315329313,
+                    0.00607642298564,
+                    0.000168528946233,
+                    0.0274331402034
+                 ],
+                 'dvar':[
+                    0.0290206372738,
+                    0.0327779129148,
+                    0.0200501643121,
+                    0.00221319519915,
+                    6.26860273769e-05,
+                    0.00980541761965
+                 ],
+                 'dmean':[
+                    0.170528307557,
+                    0.192360565066,
+                    0.126187071204,
+                    0.031432043761,
+                    0.00537649122998,
+                    0.0285477209836
+                 ],
+                 'max':[
+                    0.944636106491,
+                    0.987443983555,
+                    0.971004068851,
+                    0.404173195362,
+                    0.0544534698129,
+                    0.777507662773
+                 ],
+                 'var':[
+                    0.0980709269643,
+                    0.072200588882,
+                    0.0500059388578,
+                    0.00271857017651,
+                    6.26871405984e-05,
+                    0.00670601474121
+                 ],
+                 'mean':[
+                    0.450265616179,
+                    0.34460362792,
+                    0.196296930313,
+                    0.0317945256829,
+                    0.00625067576766,
+                    0.0202882513404
+                 ]
+              }
+           },
+           ```
+        """
+        return load_extractor(self.path)["metadata"]["rhythm"]
+
+    def to_jams(self):
+        """Jams: the track's data in jams format"""
+        return jams_utils.jams_converter(
+            metadata={'features': load_extractor(self.path)}
+        )
+
+
+def load_extractor(path):
+    """Load a AcousticBrainz Dataset json file with all the features and metadata.
+
+    Args:
+        path (str): path to features and metadata path
+
+    Returns:
+        y (np.ndarray): the mono audio signal
+        sr (float): The sample rate of the audio file
+
+    """
+    if not os.path.exists(path):
+        raise IOError("path {} does not exist".format(path))
+
+    with open(path) as json_file:
+        meta = json.load(json_file)
+    return meta
+
+
+def download(data_home=None, force_overwrite=False, cleanup=True):
+    """Download the AcousticBrainz Dataset Dataset (annotations).
```

Comment:
Genre?

Reply:
nit - In this dataset this message is no needed